CONTRIBUTING.md

```diff
@@ -60,8 +60,10 @@ This project welcomes non-code contributions, too! The following types of contri
 - **Ideas**: Participate in an issue thread or start your own to have your voice heard.
 - **Bugreports**: Found a bug in LibrePCB? Just open an issue and describe how to reproduce that bug.
 - **Small Fixes**: Fix typos, clarify language, and generally improve the quality of the content.
-- **Documentation**: Create/improve documentations for end-users or for developers of LibrePCB.
-- **Translations**: Add/improve [translations for LibrePCB](https://github.com/LibrePCB/LibrePCB/tree/master/i18n).
+- **Documentation**: Create/improve [documentations for users](https://github.com/LibrePCB/librepcb-doc)
+  or developers of LibrePCB.
+- ~~**Translations**: Add/improve [translations for LibrePCB](https://github.com/LibrePCB/LibrePCB/tree/master/i18n).~~
+  *(not yet applicable)*
 - **Website**: Improve our [website](http://librepcb.org) which is hosted in the repository [LibrePCB/LibrePCB.github.io](https://github.com/LibrePCB/LibrePCB.github.io).
 - **Sharing**: Speak about LibrePCB with your friends and colleagues, or write about it in the internet!
 - **Donations**: Donate to the LibrePCB developers with Bitcoin: [1FiXZxoXe3px1nNuNygRb1NwcYr6U8AvG8](bitcoin:1FiXZxoXe3px1nNuNygRb1NwcYr6U8AvG8)
```

Comment:
If you're at it, you could also add Patreon here.

Reply:
Good point -> done.